include/config/pokemon.h

```diff
@@ -7,6 +7,9 @@
 #define P_UPDATED_ABILITIES         GEN_LATEST  // Since Gen 6, certain Pokémon have their abilities changed.
 #define P_UPDATED_EGG_GROUPS        GEN_LATEST  // Since Gen 8, certain Pokémon have gained new egg groups.
 
+// Evolution settings
+#define P_FRIENDSHIP_EVO_THRESHOLD   160  // Since Gen 8, Pokémon that evolve by friendship evolve at or above 160 friendship instead of 220.
```

Comment:
Most configs use the `GEN_x` system, so let's stick with that.

Reply:
Not necessarily. I think this is a good config to have fine tuned control over, and it makes code cleaner.

Reply:
I think someone else should weight in

Reply:
I agree with a `GEN_x` config. There is a config for gems and type boosting items and I would treat this the same. People can still fine tune regardless.

Reply:
Alright, I'll change it then